package.json

```diff
@@ -5,7 +5,7 @@
   "description": "the autodesk ui library",
   "main": "index.js",
   "scripts": {
-    "test": "lerna run test; lerna run lint"
+    "test": "lerna run lint && lerna run test"
```

Comment:
By putting the semi-colon instead of && we were always returning 0 as the exit code instead of the correct (0 or 1) which was causing the build to always be green.

cc @nfiniteset